test/jdk/java/text/Format/NumberFormat/CurrencySymbols.properties

```diff
@@ -1,3 +1,28 @@
+#
+# Copyright (c) 2023, Oracle and/or its affiliates. All rights reserved.
+# DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+#
+# This code is free software; you can redistribute it and/or modify it
+# under the terms of the GNU General Public License version 2 only, as
+# published by the Free Software Foundation.  Oracle designates this
+# particular file as subject to the "Classpath" exception as provided
+# by Oracle in the LICENSE file that accompanied this code.
```

Comment:
Classpath exception is not applicable to tests.

Reply:
Did not know that, thank you. Made the fixes for this and your previous comment.